src/main/java/org/jabref/gui/desktop/JabRefDesktop.java

```diff
@@ -315,26 +313,5 @@ public static void openBrowserShowPopup(String url, DialogService dialogService,
             dialogService.showErrorDialogAndWait(couldNotOpenBrowser, couldNotOpenBrowser + "\n" + openManually + "\n" + copiedToClipboard);
         }
     }
-
-    /**
-     * Moves the given file to the trash.
-     *
-     * @throws UnsupportedOperationException if the current platform does not support the {@link Desktop.Action#MOVE_TO_TRASH} action
-     * @see Desktop#moveToTrash(File)
-     */
-    public static void moveToTrash(Path path) {
-        NATIVE_DESKTOP.moveToTrash(path);
-    }
-
-    public static boolean moveToTrashSupported() {
-        return NATIVE_DESKTOP.moveToTrashSupported();
-    }
-
-    public static Path getApplicationDirectory() {
-        return NATIVE_DESKTOP.getApplicationDirectory();
-    }
-
-    public static Path getFulltextIndexBaseDirectory() {
-        return NATIVE_DESKTOP.getFulltextIndexBaseDirectory();
-    }
 }
```

Comment:
Please keep the methods here. `OS.getNativeDesktop()` may only be used by certain classes. See `org.jabref.architecture.MainArchitectureTest#nativeDesktopIsRestricted`.

Reply:
For me this is a code smell because we essentially have duplicate methods now. Before the mentioned PR we didn't have this architecture check